packages/react/src/auto/hooks/useModelActionsForTable.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from "react";
+import deepEqual from "react-fast-compare";
+import { humanizeCamelCase } from "../../utils.js";
+
+export type ModelActionDetails = {
+  apiIdentifier: string;
+  operationName: string;
+  isDeleter: boolean;
+  isBulk: boolean;
+  acceptsModelInput: boolean;
+  variables: any;
+};
+
+/**
+ * Returns a list of ModelActionDetail objects for given model
+ */
+const useModelActionsForTable = (props: { model: any }) => {
```

Comment:
These feel more like helpers than hooks -- I think these could be just utils functions

Reply:
Updated this helper such that it no longer has a hook-like name